build.gradle.kts

```diff
@@ -28,6 +29,23 @@ detekt {
 
 }
 
+koverReport {
+    val excludedFiles = listOf(
+        "*.BuildConfig.*",
+        "*.BuildConfig",
+        // Jetpack Compose
+        "*.ComposableSingletons*",
+        "*.*\$*Preview\$*",
+        "*.ui.preview.*",
+    )
+
+    filters {
+        excludes {
+            classes(excludedFiles)
+        }
+    }
+}
```

Comment:
@lydiasama We must follow the latest config of Kover 0.7.0 or later by moving this config to the app level instead of the root level. Please re-check the latest sample here https://github.com/Kotlin/kotlinx-kover/tree/v0.7.0/examples/android/multiplatform, and this migrate PR https://github.com/nimblehq/android-templates/pull/501